test/evmc/mocked_host.hpp

```diff
@@ -92,15 +92,10 @@ class MockedHost : public Host
         /// The address of the beneficiary account.
         address beneficiary;
 
-        /// The balance of the self-destructed account.
-        uint256be balance;
-
         /// Equal operator.
         bool operator==(const selfdestuct_record& other) const noexcept
         {
-            return selfdestructed == other.selfdestructed &&
-                beneficiary == other.beneficiary &&
-                balance == other.balance;
+            return selfdestructed == other.selfdestructed && beneficiary == other.beneficiary;
```

Comment:
I added this check in https://github.com/ethereum/solidity/pull/11061/files in order to include "balance" for comparison. Is this redundant?